CIP-0048/README.md

```diff
@@ -0,0 +1,249 @@
+---
+CIP: 48?
+Title: 721 References and Payloads
+Authors: Jack (ada9000, https://forum.cardano.org/u/jack7e/, @ada9000_ twitter)
```

Comment:
```suggestion
Authors:
    - Jack <some@real.email.example.org>
```
There's no means to embed URLs in header text, which makes social links impossible.  Even if getting an anonymous email to use here, we need something that looks like an email address if only so our web site builders work properly.  This is always a list even if having only one member.